psi-ms.obo

```diff
@@ -22118,6 +22119,75 @@ name: Orbitrap Ascend
 def: "Thermo Scientific Orbitrap Ascend mass spectrometer with Tribrid architecture consisting of quadrupole mass filter, linear ion trap and Orbitrap mass analyzers." [PSI:PI]
 is_a: MS:1000494 ! Thermo Scientific instrument model
 
+[Term]
+id: MS:1003357
+name: ANN-SoLo
+def: "ANN-SoLo (Approximate Nearest Neighbor Spectral Library) is a spectral library search engine for fast and accurate open modification searching. ANN-SoLo uses approximate nearest neighbor indexing to speed up open modification searching by selecting only a limited number of the most relevant library spectra to compare to an unknown query spectrum. This is combined with a cascade search strategy to maximize the number of identified unmodified and modified spectra while strictly controlling the false discovery rate and the shifted dot product score to sensitively match modified spectra to their unmodified counterpart." [doi:10.1021/acs.jproteome.8b00359, doi:10.1021/acs.jproteome.9b00291]
+is_a: MS:1001456 ! analysis software
+
+[Term]
+id: MS:1003358
+name: XCorr rank
+def: "The rank of this PSM relative to all other PSMs involving this spectrum, when sorting by the XCorr score." [PSI:MS]
+is_a: MS:1001143 ! PSM-level search engine specific statistic
+relationship: has_value_type xsd:double ! The allowed value-type for this CV term
+
+[Term]
+id: MS:1003359
+name: exact p-value
+def: "A p-value for the XCorr score, calculated using dynamic programming." [PMID:24895379]
+is_a: MS:1001143 ! PSM-level search engine specific statistic
+relationship: has_value_type xsd:double ! The allowed value-type for this CV term
+
+[Term]
+id: MS:1003360
+name: refactored XCorr
+def: "A modified version of the XCorr score that is made amenable to dynamic programming calculation of p-values by changing a max operation to a sum." [PMID:30221945]
+is_a: MS:1001143 ! PSM-level search engine specific statistic
+relationship: has_value_type xsd:double ! The allowed value-type for this CV term
+
+[Term]
+id: MS:1003361
+name: res-ev score
+def: "The residue-evidence (res-ev) score measures the quality of a match between a peptide and observed spectrum using a method similar to XCorr, but considering all pairs of observed peaks. (PMID:30221945)" [PSI:MS]
```

Comment:
Last little quibble is to put the PMID CURIE in square brackets like the others instead of in parens at the end of the definition, unless there a reason for doing it differently?

Reply:
Yes, we previously used it like so:
```
id: MS:1000598
name: electron transfer dissociation
def: "A process to fragment ions in a mass spectrometer by inducing fragmentation of cations (e.g. peptides or proteins) by transferring electrons from radical-anions." [DOI:10.1073/pnas.0402700101, PMID:15210983, PSI:MS]
synonym: "ETD" EXACT []
```